homeassistant/components/enigma2/manifest.json

```diff
@@ -2,8 +2,9 @@
   "domain": "enigma2",
   "name": "Enigma2 (OpenWebif)",
   "codeowners": ["@autinerd"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/enigma2",
   "iot_class": "local_polling",
   "loggers": ["openwebif"],
-  "requirements": ["openwebifpy==4.0.0"]
+  "requirements": ["openwebifpy==4.0.2"]
```

Comment:
Please bump in separate PR

Reply:
The bump was already done, just forget to rebase.